packages/blade/src/components/Badge/Badge.tsx

```diff
@@ -131,7 +131,7 @@ const _Badge = ({
 
   return (
     <BaseBox
-      display="flex"
+      display={(isReactNative() ? 'flex' : 'inline-flex') as never}
```

Comment:
as `never` 😢 

@anuraghazra iska kuch karna padega long term ke liye. Not a PR blocker but we need to figure out something soon. Upgrade RN me bhi similar issues the